lib/urlwatch/filters.py

```diff
@@ -183,6 +183,20 @@ def filter(self, data, subfilter=None):
         return ical2text(data)
 
 
+class JsonFormatFilter(FilterBase):
+    """Convert to formatted json"""
+
+    __kind__ = 'formatted-json'
```

Comment:
Maybe just call this `format-json` to be more in line with the "verb"-style filter names (what does it do?) vs the "noun"-style filter name (what is the output?).

Reply:
`format-json` sounds much better and it's also shorter to write. Changed.